components/AUPage/Content.jsx

```diff
@@ -0,0 +1,256 @@
+  },
+  {
+    id: '',
+    name: 'Kryptoplanet',
+    iconFileName: '/au-page/icons/kryptoplanet.png',
+  },
+  {
+    id: '',
+    name: 'Polywrap',
+    iconFileName: '/au-page/icons/polywrap.png',
+  },
+];
+
+const mediumIcons = [
+  {
+    id: '',
+    name: 'Vana',
+    iconFileName: '/au-page/icons/vana.svg',
+  },
+  {
+    id: '',
+    name: 'Myshell',
+    iconFileName: '/au-page/icons/myshell.png',
+  },
+  {
+    id: '',
+    name: 'Capx',
+    iconFileName: '/au-page/icons/capx.svg',
+  },
+  {
+    id: '',
+    name: 'Agentcoin',
+    iconFileName: '/au-page/icons/agentcoin.svg',
+  },
+  {
+    id: '',
+    name: 'Layer',
+    iconFileName: '/au-page/icons/layer.svg',
+  },
+  {
+    id: '',
+    name: 'Phala',
+    iconFileName: '/au-page/icons/phala.svg',
+  },
+  {
+    id: '',
+    name: 'Creator',
+    iconFileName: '/au-page/icons/creator.png',
+  },
+  {
+    id: '',
+    name: 'Gensyn',
+    iconFileName: '/au-page/icons/gensyn.svg',
+  },
+  {
+    id: '',
+    name: 'Spectral',
+    iconFileName: '/au-page/icons/spectral.png',
+  },
+  {
+    id: '',
+    name: 'Edge Network',
+    iconFileName: '/au-page/icons/edge-network.png',
+  },
+  {
+    id: '',
+    name: 'Agentops',
+    iconFileName: '/au-page/icons/agentops.png',
+  },
+];
+
+const smallIcons = [
+  {
+    id: '',
+    name: 'Nevermined',
+    iconFileName: '/builders/nevermined.png',
+  },
+  {
+    id: '',
+    name: 'Signature Ventures',
+    iconFileName: '/au-page/icons/signature-ventures.png',
+  },
+  {
+    id: '',
+    name: 'Keyko',
+    iconFileName: '/au-page/icons/keyko.png',
+  },
+  {
+    id: '',
+    name: 'The Indexing Company',
+    iconFileName: '/au-page/icons/indexing-company.png',
+  },
+  {
+    id: '',
+    name: 'Naptha AI',
+    iconFileName: '/builders/naptha-ai.png',
+  },
+  {
+    id: '',
+    name: 'Wayfinder',
+    iconFileName: '/au-page/icons/wayfinder.png',
+  },
+  {
+    id: '',
+    name: 'Mode',
+    iconFileName: '/chains/mode.svg',
+  },
+  {
+    id: '',
+    name: 'Biconomy',
+    iconFileName: '/au-page/icons/biconomy.png',
+  },
+  {
+    id: '',
+    name: 'Newcoin',
+    iconFileName: '/au-page/icons/newcoin.png',
+  },
+  {
+    id: '',
+    name: 'Kaito',
+    iconFileName: '/au-page/icons/kaito.png',
+  },
+  {
+    id: '',
+    name: 'Coophive',
+    iconFileName: '/au-page/icons/coophive.png',
+  },
+  {
+    id: '',
+    name: 'Voiceflow',
+    iconFileName: '/au-page/icons/voiceflow.png',
+  },
+  {
+    id: '',
+    name: 'Story',
+    iconFileName: '/au-page/icons/story.png',
+  },
+];
+
+const Sponsors = () => (
+  <div className="mx-auto max-w-[1000px] max-sm:py-8 max-sm:px-6">
+    <div className="grid grid-cols-2 sm:grid-cols-3 gap-x-24 gap-y-4 mb-20">
+      {largeIcons.map((icon) => {
+        const { id, name, iconFileName } = icon;
+        return (
+          <div
+            key={id}
+            className="grayscale flex max-w-[200px] justify-center items-center"
```

Comment:
as I understood from Figma, each second image in a row should be in the center, each third - aligned to the right. you can use the index from array and provide needed classes (made by chatgpt :D hope, works)

```
className={`grayscale flex max-w-[200px] justify-center items-center ${
        (index + 1) % 3 === 2 ? 'justify-self-center' : ''
      } ${(index + 1) % 3 === 0 ? 'justify-self-end' : ''}`}
```